### DISCUSSION TITLE
What does the SD icon mean?

### DISCUSSION THREAD

**Discussion by t33st33r** [Universal-Team/Universal-Updater#119] [Q&A]
What's the meaning of the SD icon in "Available Downloads" section? It's next (right side) to the filename.

It is gone if I touch it. Once gone, touching the same area again doesn't make it come back.

**Comment by Epicpkmn11** [comment]
It indicates that the app is installed, which is used for showing the update tracking on the top screen and the "Add selection to queue" button in the search menu. Tapping it makes it go away as a way to indicate you uninstalled the app, since UU doesn't have proper uninstall tracking as that's rather difficult with the variety of ways an app can be installed.
